packages/vite/src/node/preview.ts

```diff
@@ -16,6 +16,11 @@ import { proxyMiddleware } from './server/middlewares/proxy'
 import { printServerUrls } from './logger'
 import { resolveHostname } from './utils'
 
+/**
+ * Starts the Vite server in preview mode, to simulate a production deployment
+ * @param config the resolve Vite config
+ * @param serverOptions what host and port to use
+ */
```

Comment:
This is a good start 👍 
But I would also like a part in the Vitepress documentation. @patak-js Or don't we have such a section?

Reply:
It should be documented after `build` here https://vitejs.dev/guide/api-javascript.html#build, but again, I think we should wait to do so as we don't know if we are going to be able to merge this PR.

Reply:
Or maybe it should be exposed as `@alpha` or `@beta` for a period of time and not documented quite yet so that the interface can still be changed if we discover some reason it needs to be tweaked? I'm okay having SvelteKit rely on an API that might change, but understand the potential hesitancy around making public APIs that must be maintained, so if there's some middle ground where we can use it but don't commit Vite to it that would be fine with me

Reply:
I think we already used `@experimental` did we? ... Yes we did, checked it (has a typo :laughing:)
But we also use `@alpha` on one part.

All three are possible I think

- https://tsdoc.org/pages/tags/experimental
- https://tsdoc.org/pages/tags/alpha
- https://tsdoc.org/pages/tags/beta